client/metric.go

```diff
@@ -41,6 +41,7 @@ func (c *httpHealthMetrics) startObserve(ctx context.Context) {
 		select {
 		case <-ctx.Done():
 			return
+		default:
```

Comment:
I would approve and merge this in a separate PR ;)

Reply:
resolved in #488